azkaban-common/src/test/java/azkaban/project/ProjectSpecLoader.java

```diff
@@ -0,0 +1,14 @@
+package azkaban.project;
```

Comment:
Please add a copyright notice to all new files.
Is there a tool to check and auto update copyright notices?